src/components/common/PeopleAndRoles/CurrentDirectors.vue

```diff
@@ -43,20 +43,23 @@
 
             <!-- Mailing Address -->
             <td class="px-0">
-              <base-address class="director-detail" :address="orgPerson.mailingAddress" />
+              <MailingAddress class="director-detail" :address="orgPerson.mailingAddress" />
             </td>
 
             <!-- Delivery Address -->
             <td class="px-0">
-              <p v-if="isSame(orgPerson.mailingAddress, orgPerson.deliveryAddress)"
-                class="director-detail">Same as Mailing Address
+              <p
+                v-if="isSame(orgPerson.mailingAddress, orgPerson.deliveryAddress, ['id'])"
```

Comment:
Ignore `id` from comparison. (Although the UI stores separate mailing and delivery addresses, it doesn't care about id for display purposes. It's used by the API.)